bpm.py

```diff
@@ -1,11 +1,11 @@
-from PyQt4.QtCore import QTimer
-from PyQt4 import QtCore
+from PyQt5.QtCore import QTimer
+from PyQt5 import QtCore
 from recorder import *
 from time import perf_counter
 
 class AudioAnalyzer:
-    min_bpm = 60
-    max_bpm = 180
```

Comment:
Did this, because the algorithm can not detect the difference between 60 and 120 BPM (etc).